extensions/classifiers/LDAStringClassifier/LDAStringClassifier.py

```diff
@@ -627,9 +690,7 @@ def validate(self, classifier_data):
             '_w_dp',
             '_l_dp',
             '_c_dl',
-            '_c_lw'
-        ]
-        list_of_list_bool_properties = [
+            '_c_lw',
             '_b_dl'
```

Comment:
It was made a boolean property check by mistake. The data is not really expected to be booleans. It was always stored as 0/1s matrix.